src/PhpSessionPersistence.php

```diff
@@ -370,6 +370,10 @@ private function responseAlreadyHasCacheHeaders(ResponseInterface $response) : b
 
     private function getCookieLifetime(SessionInterface $session) : int
     {
+        if (empty($session->toArray())) {
```

Comment:
can use truthy check: 

```suggestion
        if (! $session->toArray()) {
```

Reply:
I was under the impression that truthy check for an array was against laminas code style. Am I confusing with another project ?

Reply:
previously, I proposed pr on some repos for eliminate count() for empty check on arrays, with empty() check, and @Ocramius review it to use truthy check instead. The changes already applied on some repos for that. I think that can be used for consistency.

Reply:
Thank for letting me know. This is also my preferred style, so I'm glad it's accepted.